.circleci/deployment/workflows.yml

```diff
@@ -71,3 +71,34 @@
             branches:
               only:
                 - master
+
+  dev-apps-fs-upgrade:
+    when:
+      << pipeline.parameters.upgrade_dev_apps_fs >>
```

Comment:
I can't find it -- is there an evaluation where we can set these to True instead of the default False? The testing steps mention running a CLI alias `tdrs-deploy` but I don't see commands.sh updated here? Are we missing some commits?

Reply:
There is still some manual intervention required since it isnt really feasible to monitor when CF releases a new OS. But the script updates all the apps that dont get updated by the deploy scripts.